etc/fixtures/migrations/migrate-insert-discussion-and-supporting-keys.js

```diff
@@ -0,0 +1,66 @@
+const usage = `
```

Comment:
Would you like to add the migration script to be double-checked as a part of the PR?  I can pull up an old PR and use similar instructions to verify it if it's alright.

Reply:
https://github.com/uab-cgds-worthey/rosalution/pull/154#issue-2038395590

Let me provide that old pull request for the migration script. This will need to happen on the development cluster after this feature branch goes into main.

I can add it as a part of this review, but there should not have been any changes in subsequent commits to this script/structure.

Reply:
Dang, I totally forgot I PRed the migration already.  This is sufficient for me.

Reply:
Just for the sake of completeness, I ran the migration script again on the current data:

<img width="684" alt="Screenshot 2024-02-08 at 3 17 24 PM" src="https://github.com/uab-cgds-worthey/rosalution/assets/5799712/e7303c3e-432f-489f-82de-a84178ede437">

![Screenshot 2024-02-08 at 2 32 55 PM](https://github.com/uab-cgds-worthey/rosalution/assets/5799712/fd96e253-5c71-4ec6-b740-3e98eca27b92)